extensions/infinispan-embedded/runtime/src/main/java/io/quarkus/infinispan/embedded/runtime/InfinispanEmbeddedRuntimeConfig.java

```diff
@@ -9,7 +9,7 @@
 /**
  * @author wburns
  */
-@ConfigRoot(name = "infinispan-embedded", phase = ConfigPhase.RUN_TIME)
+@ConfigRoot(name = "infinispan-embedded", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
Do we really want this change? As configurations knobs won’t be overridable at runtime.

Reply:
There's just this one property in that config and IMO it doesn't make sense to override that one at runtime. So logically I think this should be a build-time-only config, but it had to be made available at run-time because of how Infinispan works - it is being read by Infinispan at runtime.

Reply:
At the very least we would need input from @wburns before making this change

Reply:
👍 for separating this concern to the other issue. I have responded to why it is RUN_TIME for now at https://github.com/quarkusio/quarkus/issues/6392#issuecomment-575227225